v2/src/modules/address/components/TableRowNftTransfers.vue

```diff
@@ -1,34 +1,36 @@
 <template>
     <!--
-               ========================
-                 Desktop View
-               =========================
-        -->
-    <app-table-row v-if="!mdAndDown" row-align="center">
-        <v-col :cols="props.isOverview ? 4 : 3">
-            <v-row class="ma-0 flex-nowrap" align="center">
-                <img :src="tokenImg" alt="" height="41" width="41" class="mr-2 rounded-circle" />
-                <div style="display: grid">
-                    <router-link
-                        v-if="props.transfer.tokenInfo.name !== '' || props.transfer.tokenInfo.symbol"
-                        :to="`/token/${props.transfer.contract}`"
-                        class="text-textPrimary text-ellipses"
-                    >
-                        <p v-if="props.transfer.tokenInfo.name" class="text-ellipses">{{ props.transfer.tokenInfo.name }}</p>
-                        <p v-else class="text-uppercase caption text-ellipses">N/A</p>
-                    </router-link>
-                </div>
-            </v-row>
+    ========================
+        Desktop View
+    =========================
+    -->
+    <app-table-row v-if="!smAndDown" row-align="center">
+        <v-col :cols="props.isOverview ? 4 : 3" class="d-flex align-center">
+            <token-nft-img
```

Comment:
Currently the loading behaves weirdly. Probably cause we had to load the meta data after the initial data is loaded, so we see two different loading animation

Reply:
fixed in next commit :)